controllers/component_build_controller_test.go

```diff
@@ -2178,6 +2273,7 @@ var _ = Describe("Component initial build controller", func() {
 		})
 
 		It("should use the global build bundle", func() {
+			deleteBuildPipelineRunSelector(defaultSelectorKey)
```

Comment:
We do not delete pipeline run selector for the test above, but delete it for this test. Why?
It would be better to avoid fixing order of tests here.